### ISSUE TITLE
vaadin grid - collapse column menu broken when using with many columns

### ISSUE THREAD

**Issue by vm-bio** [vaadin/framework#9569]
Hi there,
this is my 1st bug report for vaadin so please be not too strong with me if I did sth. wrong.

There is an issue with vaadin grids when using it with many columns in IE11 (Firefox works as expected):
If the menu for hiding columns is longer then the window allows, which means that the scrollbar is added, you can't select entries below the 1st page. The list "jumps back" as soon as you move the mouse over an item.
This is reproducable with the vaadin sampler https://demo.vaadin.com/sampler/#ui/grids-and-trees/grid/features
1) shorten the browser window to ensure that the opened "hamburger menu" for hiding columns doesn't fit into the window to force the scrollbar to be shown
2) click the "hamburger menu"
3) scroll down a bit (scrollbar moves as expected)
4) (try to) move the mouse over a specific entry
-> menu scrolls back to start; you can't select the entry you want.

If you move the selected entry using the cursor keys the selected entry moves out of the visible frame - probably to the "wanted entry".

Ah - for the protocol:
I'm using Vaadin 7.7.9 (no I can't port to Vaadin 8 due to the plugins used) with IE 11.0.9600.18665 Updateversion 11.0.42 (and this is not changeable).

I'd be glad if this could be fixed soon ...

Bye, Volker

**Comment by TatuLund** [comment]
Yes. The issue is easy to reproduce in IE11. It is not happening with Edge. This gives a hint that it is actually a bug in IE11 that has been fixed in Edge. Having said that, it could be possible to have some sort of workaround for this. There are some related discussion here: https://github.com/noraesae/perfect-scrollbar/issues/160

**Comment by Ansku** [comment]
Duplicate of #7874
